routing/localchans/manager.go

```diff
@@ -180,6 +216,87 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 	return failedUpdates, nil
 }
 
+// createEdge recreates an edge and policy from an open channel in-memory.
+func (r *Manager) createEdge(channel *channeldb.OpenChannel) (
```

Comment:
would be great to have a unit test for this - meanwhile have you tested this cli out locally? Think it's not suitable for itest as it's hard to create this edge case.

Reply:
I'll cook a test. 

I haven't tested this out yet, I was really looking for your review here before we deploy this in production (which is where the issue occurs)